labgrid/driver/usbstoragedriver.py

```diff
@@ -100,14 +100,18 @@ def write_image(self, filename=None, mode=Mode.DD, partition=None, skip=0, seek=
                 "conv=fdatasync"
             ]
         elif mode == Mode.BMAPTOOL:
+            mf_bmap = ManagedFile("{}.bmap".format(filename), self.storage)
+            mf_bmap.sync_to_resource()
+
             if skip or seek:
                 raise ExecutionError("bmaptool does not support skip or seek")
             self.logger.info('Writing %s to %s using bmaptool.', remote_path, target)
             args = [
                 "bmaptool",
                 "copy",
+                "--bmap={}".format(mf_bmap.get_remote_path()),
                 "{}".format(remote_path),
-                "{}{}".format(target),
+                "{}".format(target),
```

Comment:
This is already fixed by #659. @Emantor care to merge that?

Reply:
Done.